web/LICENSE.txt

```diff
@@ -0,0 +1,18 @@
+The MIT License (MIT)
```

Comment:
Any way we could refer to the "main" license file instead of duplicating one here (and in the sfe package)?

Reply:
The licenses for the front-end and the back-end are different.  Jens has also expressed a desire to move to a model of independently shippable apps for verticals (user-settings, user-library, admin, flow, etc), and each would need its own copy of the license.